### DISCUSSION TITLE
Read assets for guest not working

### DISCUSSION THREAD

**Discussion by leguennecj** [requarks/wiki#4589] [Error / Bug Report] [answered]
Hello,

I have a problem with guest permissions. Indeed if you have a look on that page : https://wiki.olenergies.com/fr/Notices

you will not see the medias (images) but I have activated permissions to read assets for guests.
https://ibb.co/2Sbtn9T
https://ibb.co/mTnR5ZW
https://ibb.co/Jv0mNjQ

Can you help me to accept guest to see that page exclusively with assets but not all my wiki ?

Indeed if I use the page rule : "read:page" / "read:assets" --> page start with " / " it works.
https://ibb.co/4pFRfpn
https://ibb.co/n3GGDFz

Thanks for you support

Julien

**Comment by leguennecj** [comment, marked as answer]
I have finally find. 
Need to add a rule using read:assets starting with path /

**Comment by mtghr** [comment]
Thanks that worked for me

still seems like a bug to me, why would i grant Read:assest to / ?

if some one has a direct link to an aseest they will be able to see it !

**Reply by YerikYuan** [reply]
Yes! Hope no one scans the root directory of the server.
It is a bug indeed!
The current setting is also ambiguous and hard to understand.
Maybe wiki.js could learn from mediawiki and confluence wiki.

**Comment by gdevenyi** [comment]
I think I would propose the solution here that assets inserted into a page should (virtually?) inherit the path of the page they're in, so page rules make more sense.
